src/battle_game/app/app.cpp

```diff
@@ -3,6 +3,10 @@
 #include "battle_game/core/object.h"
 #include "battle_game/graphics/util.h"
 
+const int TotalBuffNumber = 1;
+const char *BuffDisplay[] = {"Stunning: %.1f"};
+int BuffID[] = {0};
```

Comment:
```suggestion
namespace {
  const int TotalBuffNumber = 1;
  const char *BuffDisplay[] = {"Stunning: %.1f"};
  int BuffID[] = {0};
}
```

variable names should be in lower_case